src/main/resources/org/springframework/integration/config/xml/spring-integration-kafka-1.1.xsd

```diff
@@ -0,0 +1,708 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No reason in the separate file: just rename to the new version is enough.
See how we live with that in the SI Core.